### ISSUE TITLE
webFrame.setZoomLevel impacts all webFrames in the same page.

### ISSUE THREAD

**Issue by burtonator** [electron/electron#13971]
* Electron Version: 3.0.0-beta4
* Operating System (Platform and Version): Ubuntu 18.0.4
* Last known working Electron version: unknown

**Expected Behavior**

setZoomLevel should only effect the webFrame it was executed upon.

**Actual behavior**

It actually impacts them all..

Basically. if you call webFrame.setZoomLevel all webFrames are impacted, even if it's just an iframe.  

I'm calling webFrame.getFrameForSelector('iframe').setZoomLevel(2)

and the entire page has its zoom level changed.

I'll eventually create a repo for this if you guys don't have a simple fix.

**Comment by ckerr** [comment]
Yes, a repo for this forked from electron-quick-start would be great. Thank you for offering!
